test/unit/test-runner-test.coffee

```diff
@@ -17,7 +17,7 @@ TestRunner = proxyquire '../../lib/test-runner', {
 }
 
 ABAO_IO_SERVER = 'http://abao.io'
-TEST_SERVER = 'http://localhost:3000'
+SERVER = 'http://localhost:3000'
```

Comment:
Move what into the RAML file? The `SERVER` variable value? This file was about testing the TestRunner code, not about #25 . Tried to keep most of what was there logically intact. Still not sure about what the `ABAO_IO_SERVER` is supposed to represent as the hostname won't resolve -- or was that the point?

Regardless of this variable, does beta5 meet your needs?